cmd/zoekt-archive-index/main.go

```diff
@@ -152,6 +152,11 @@ func do(opts Options, bopts build.Options) error {
 	}
 	defer a.Close()
 
+	ig, err := newIgnoreMatcher(opts.Archive, ignoreOptions{strip: opts.Strip})
```

Comment:
I didn't add a flag to `zoekt-archive-index` (and consequently neither to `zoekt-sourcegraph-indexserver`) because I assume we want to switch the feature on for all customers. However, customers without a `sourcegraphignore` would pay a small performance fee during indexing.  Is there a better way to check for the existence of a `sourcegraphignore` file without looping through all files?